app/helpers/app_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module AppHelper
+  def release_specific_channel_pattern(app)
```

Comment:
Could we move this to `ApplicationHelper` pls? I'm trying to get rid of domain-level helpers as much as possible and dumping them in one helper (because that's what helpers do anyway).